### DISCUSSION TITLE
Implementing a Tab Merging Feature to Enhance the Windows 11 Explorer User Experience

### DISCUSSION THREAD

**Discussion by nointerleaving** [ramensoftware/windhawk-mods#177] [Mod ideas]
Dear Windows 11 development team,

I would like to suggest a new feature for the Windows 11 Explorer that would greatly enhance its usability. My suggestion is to implement a tab merging function that allows users to combine multiple tabs or windows into a single window.

This feature would be especially useful for users who frequently work with multiple windows or tabs open at the same time. By allowing users to merge tabs, they would be able to declutter their desktop and have a more organized workspace. Additionally, this feature would save time and effort by eliminating the need to switch between different tabs or windows.

I believe that this feature would be a valuable addition to the Windows 11 Explorer and would greatly enhance the overall user experience. Thank you for considering my suggestion.

Sincerely,

Inter

Text made by chat gpt

**Comment by windwhim** [comment]
有一个软件可以做到这一点
https://github.com/w4po/ExplorerTabUtility
